src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -215,38 +233,25 @@ const RuleEditor = createReactClass({
               nodes={this.props.actions}
               initialItems={actions}
               className="rule-action-list"
-              onChange={this.onActionsChange}
             />
 
             <hr />
 
             <div className="node-frequency-selector">
-              <h6>
-                {tct(
-                  'Perform these actions at most once every [frequency] for an issue.',
-                  {
-                    frequency: (
-                      <SelectInput
-                        ref="frequency"
-                        className={this.hasError('frequency') ? ' error' : ''}
-                        value={frequency}
-                        style={{width: 150}}
-                        required={true}
-                      >
-                        <option value="5">{t('5 minutes')}</option>
-                        <option value="10">{t('10 minutes')}</option>
-                        <option value="30">{t('30 minutes')}</option>
-                        <option value="60">{t('60 minutes')}</option>
-                        <option value="180">{t('3 hours')}</option>
-                        <option value="720">{t('12 hours')}</option>
-                        <option value="1440">{t('24 hours')}</option>
-                        <option value="10080">{t('one week')}</option>
-                        <option value="43200">{t('30 days')}</option>
-                      </SelectInput>
-                    ),
-                  }
+              <AlertRuleRow>
+                {t(
+                  'Perform these actions at most once every %s for an issue.',
```

Comment:
Does this output the Select field in line with the text? I wonder what this will build as with the `%s` formatting. `tct` may be what we want here but I'm not sure. Have you tried building with `SENTRY_EXTRACT_TRANSLATIONS=1` and checking the extracted translation templates?

Reply:
Leaving this as-is since the script to check it doesn't work. Assume the select field is still being ignored like before.